Create README.md to provide project description and installation instructions (#29)
kyma-project/rt-bootstrapper: README.md

[![golangci lint](https://badgers.space/github/checks/kyma-project/rt-bootstrapper/main/golangci-lint)](https://github.com/kyma-project/rt-bootstrapper/actions/workflows/lint.yaml)
[![latest release](https://badgers.space/github/release/kyma-project/rt-bootstrapper)](https://github.com/kyma-project/rt-bootstrapper/releases/latest)

# RT Bootstrapper

> This repository contains the source code for the RT Bootstrapper Kyma component used to configure Kyma runtime components on restricted markets infrastructure.

Reviewer: ```suggestion
This repository contains the source code for the RT Bootstrapper Kyma component used to configure Kyma runtime components running in markets with individual infrastructure setups.
```

> This repository contains the source code for the `RT-bootstrapper` Kyma component used to configure Kyma Runtime components on restricted markets infrastructure.

Reviewer 2: ```suggestion
This repository contains the source code for the RT Bootstrapper Kyma component used to configure Kyma runtime components on restricted markets infrastructure.
```

## Overview

RT Bootstrapper contains two functional parts:

- Kubernetes admission webhook that intercepts the creation of Pods and namespaces labeled for restricted markets. 
  It modifies the Pod specifications to include necessary configurations, modifies image paths to use the configured remote registry, and provides pull secrets with credentials.

> - Kubernetes Controller that watches for namespaces labeled for restricted markets and ensures that the secrets with required credentials are present and synchronized in those namespaces.

Reviewer: ```suggestion
- Kubernetes Controller that watches for namespaces and ensures that the secrets with required credentials are present and synchronized in those namespaces.
```

For information how to use and configure `RT-bootstrapper`, see the [user documentation](./docs/README.md).

**Note:**
> This component is implemented as part of Kyma Runtime delivery for restricted markets.  
> Installing `RT-bootstrapper` on standard BTP managed Kyma Runtime or in self-managed Kyma Runtime cluster, may cause harm to your workloads.

## Prerequisites

- A managed Kyma Runtime instance running on BTP platform.
- Access to Kyma Runtime cluster with kubeconfig.

## Installation with Kyma Control Plane

In restricted market environment, the `RT-bootstrapper` is installed and configured automatically by Kyma Control Plane on all provisioned Kyma Runtimes.

## Installation with kubectl

> Enable the `RT-bootstrapper` component in your Kyma cluster with kubectl by applying a release manifest.  

Reviewer 2: ```suggestion
To enable the RT Bootstrapper component in your Kyma cluster, apply the release manifest using kubectl:  
```
